shenfun_elasticity/solver/elastic_problem.py

```diff
@@ -14,6 +35,70 @@ def __init__(self, N, domain, elastic_law):
         self._elastic_law = elastic_law
         self._setup_problem()
 
+    def _cleanup_boundary_conditions(self):
```

Comment:
The designation _cleanup_ could be confused with an operation that deletes all boundary condition. Maybe organize or something similar is better. A short comment what this method actually would be helpful.

I would also add 
```python3
assert hasattr(self, "_processed_bcs")
```
to guarantee that the boundary condition have been specified.

Reply:
I agree, the name might lead to confusion.